astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -1399,20 +1415,19 @@ def test_data_at_range_limit(parallel, fast_reader, guess):
         pytest.skip("Catching warnings broken in parallel mode")
     elif not fast_reader:
         pytest.skip("Python/numpy reader does not raise on Overflow")
-    with pytest.warns() as warning_lines:
+    with ctx as w:
```

Comment:
No need to shorten `warning_lines` to `w`?

Reply:
Got pestered by black on this earlier; anyway the other tests are also just using `w` (and we get at most a single line).